Fix eslint errors

tomchentw opened the pull request:
This should fix #150 .

However, when I tried to run `npm run compress:extension`, it shows an error:

```
ERROR in ./src/app/containers/App.js
Module not found: Error: Cannot resolve module 'remotedev-app/lib/components/TestGenerator' in ……/redux-devtools-extension/src/app/containers
 @ ./src/app/containers/App.js 56:21-74
```

Other than that it looks good.

zalmoxisus commented:
Thanks a lot for taking time and fixing ESLint as well!

The second error (related to #147) happens because https://github.com/alexkuz/redux-devtools-inspector/pull/38 is not on npm yet.

tomchentw commented:
Thanks for merging and the great work!

I forgot to bring up this and I've added to the top comment:

---

I also added some `"prebuild` script to run `npm run lint` script so that we can somehow reduce lint error in the future.
